orga/tests/integration/components/layout/sidebar_test.js

```diff
@@ -140,6 +140,21 @@ module('Integration | Component | Layout::Sidebar', function (hooks) {
       // then
       assert.dom('.sidebar-menu__documentation-item').doesNotExist();
     });
+
+    test('it should display team for a pro organization', async function (assert) {
```

Comment:
*question*
Est-ce que le test ne devrait pas être plutôt `should display team for organization's members` ? Il me semble qu'on affichait déjà l'onglet "équipe" pour les admin des orga pro non ?

Reply:
oui, ça s'affichait déjà pour les admins, j'ai repris les intitulés existant mais c'est vrai que c'était pas lié à leur statut d'admin